RecoTracker/TkHitPairs/src/InnerDeltaPhi.cc

```diff
@@ -158,26 +255,32 @@ PixelRecoRange<float> InnerDeltaPhi::phiRange(const Point2D& hitXY,float hitZ,fl
   phicross2 = phicross1 + dphicross;
         
 
-  // compute additional delta phi due to origin radius
-  auto deltaPhiOrig = cropped_asin( theROrigin * (dHitmag-dLayer) / (dHitmag*dLayer));
-  deltaPhiOrig *= dLayer/(rLayer*cosCross);
 
   // inner hit error taken as constant
   auto deltaPhiHit = theExtraTolerance / rLayer;
 
   // outer hit error
-//   double deltaPhiHitOuter = errRPhi/rLayer; 
+  //   double deltaPhiHitOuter = errRPhi/rLayer; 
   auto deltaPhiHitOuter = errRPhi/hitXY.mag();
 
   auto margin = deltaPhi+deltaPhiOrig+deltaPhiHit+deltaPhiHitOuter ;
 
   if (thePrecise) {
     // add multiple scattering correction
+
+    /*
     PixelRecoPointRZ zero(0., theVtxZ);
     PixelRecoPointRZ point(hitXY.mag(), hitZ);
-    auto scatt = 3.f*sigma(thePtMin,zero, point, ol) / rLayer; 
-   
-    margin += scatt ;
+    auto scatt = 3.f*sigma(thePtMin,zero, point, ol); 
+    */   
```

Comment:
it is the old version. I tend to keep previous versions around for a while to compare with the new version if issues arise 
can be protected with LogDebug or similar. (you may have noticed that I have deleted "old version" in few other placed)